docs/_includes/attr-doc.adoc

```diff
@@ -65,16 +68,27 @@ You can also set the base path to images (default: _empty_), icons (default: `./
 
 When you find yourself typing the same text repeatedly, or text that often needs to be updated, consider assigning it to a document attribute and use that instead.
 
+NOTE: If you're used to XML, you'll know document attributes as User Entities.
```

Comment:
Perhaps write this as:

If you're familiar with writing in XML, you'll recognize document attributes as (custom text) entities.